genai-perf/tests/test_cli.py

```diff
@@ -932,6 +942,145 @@ def test_positive_image_input_args(self, monkeypatch, args):
         with pytest.raises(SystemExit) as excinfo:
             parser.parse_args()
 
+    def test_defaults_applied_when_sessions_enabled(self, monkeypatch):
```

Comment:
Nit: For naming, I'd prefer if the specific feature set you're testing (session) is first so that it's easier to identify the feature and sort by it. It'd also be good if you add a comment header to say that these are the session tests.

[Beyond the scope of this PR] Long-term, I think our team should break this test file into a directory of CLI test files, broken by major feature. This file is getting too lengthy and unwieldy. CC: @debermudez